### PR TITLE
Hotfix deploy

### PR DESCRIPTION
**Pull request by sujnesh**
Screenshot 2021-05-27 at 1 58 50 PM
Screenshot 2021-05-27 at 2 16 39 PM

### CODE REVIEW

**Review by skbly7 — APPROVED** [review]

File: app/controllers/participants_controller.rb [hunk 1]
```diff
@@ -115,6 +115,19 @@ def read_notification
     end
   end
 
+  def impersonate
```
Comment (skbly7) [inline app/controllers/participants_controller.rb:118]: super_admin only

File: app/admin/dashboard.rb [hunk 2]
```diff
@@ -6,8 +6,10 @@
       tab :Background_Jobs do
         render 'jobs'
       end
-      tab :Analytics_Dashboard do
-        render 'blazer'
+      if current_participant && current_participant.super_admin?
```
Comment (skbly7) [inline app/admin/dashboard.rb:9]: wouldn't direct link i.e. `aicrowd.com/blazer` open for non super admin?

File: app/models/active_admin_participant_authorization.rb [hunk 3]
```diff
@@ -0,0 +1,13 @@
+class ActiveAdminParticipantAuthorization < ActiveAdmin::AuthorizationAdapter
+  def authorized?(action, subject = nil)
+    if subject.name == "Participant"
+      if user.super_admin?
+        return true
+      else
+        return false
+      end
+    else
+      return true
```
Comment (skbly7) [inline app/models/active_admin_participant_authorization.rb:10]: Should this blanket `true` be returned only for `admin?`?

or the requests coming to this `AuthorizationAdapter` already have non-admins filtered?

File: app/views/participants/show.html.erb [hunk 4]
```diff
@@ -52,10 +52,24 @@
           <a class="dropdown-item<%= ' active' if active_tab == 'bio' %>" data-toggle='tab' href="#tab-bio" onclick="return false;">Bio</a>
           <a class="dropdown-item<%= ' active' if active_tab == 'posts' %>" data-toggle='tab' href="#tab-posts" onclick="return false;">Posts</a>
           <a class="dropdown-item<%= ' active' if active_tab == 'achievements' %>" data-toggle='tab' href="#tab-achievements" onclick="return false;">Achievements</a>
+          <% if true_participant.present? && true_participant.super_admin? %>
```
Comment (skbly7) [inline app/views/participants/show.html.erb:55]: `true_participant`  🤔 ? Did you mean `current_participant`?

File: app/models/active_admin_participant_authorization.rb (hunk 3 above)
Comment (sujnesh) [reply app/models/active_admin_participant_authorization.rb:10]: They've admin filter in the routes file itself

**Pull request merged**